cypress/e2e/flarumTest.cy.js

```diff
@@ -0,0 +1,53 @@
+describe('template spec', () => {
+
+  beforeEach(() => {
+    cy.loadTestDataFromJson();
+    cy.visit('/')
+  })
+
+  it('testCase1', () => {
```

Comment:
I understand that this is a practice task, but please avoid using such naming. Consider using a meaningful name that describes what the test checks, for example, "profile bio info can be changed". 
The same for the 'describe' message.